### PR TITLE
Countries

### PR DESCRIPTION
**Pull request by el-dav**
PR Summary:

New package called 'countries'

A base CountrySymbol component (similar to Icon) which is used as the wrapper for all the CountrySymbol components
Imported all the country symbol SVGs from figma

Created a script to:
- Generate the CountrySymbol components from the SVGs
- Optimise the svgs of the resulting CountrySymbol components
- Create an index file to export all the CountrySymbol components
- create a lazyMap for use in LazyCountrySymbol component
- Make the ID for each <mask> in an svg unique

Ran generation script

Created LazyCountrySymbol component which allows consumers to change the symbol using the code prop and uses code-splitting to (hopefully) reduce bundles size

Created relevant stories

Added tests

Added documentation

### CODE REVIEW

**Comment by el-dav** [comment]
Moved this to a new PR to see if chromatic build would work and cleaned up commits

**Review by Fercas123 — APPROVED** [review]

**Review by joshwooding — CHANGES_REQUESTED** [review]
Some minor changes needed on docs structure

File: packages/countries/stories/country-symbol.doc.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+We've made the symbols circular as the traditional flag shape can't easily be interchanged with other elements, such as icons or avatars. This also standardizes the shape of each flag, even when they have different proportions.
+
+The Country Symbols are stylized and representative—so users are more likely to recognize them at smaller sizes, and they maintain their character within the circular form.
+
+<Canvas>
+  <Story id="country-symbols-country-symbol--salt-country-symbol" />
+</Canvas>
+
+## Size
```
Comment (Fercas123) [inline packages/countries/stories/country-symbol.doc.stories.mdx:30]:
```suggestion
## Features

### Size
```

File: packages/countries/stories/country-symbol.doc.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+To change the size of a Country Symbol, use the `size` prop to multiply the base size.
+
+<Canvas>
+  <Story id="country-symbols-country-symbol--country-symbol-multiple-sizes" />
+</Canvas>
+
+## Usage
```
Comment (Fercas123) [inline packages/countries/stories/country-symbol.doc.stories.mdx:38]:
```suggestion
### Usage
```

File: packages/countries/stories/country-symbol.doc.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+`import { GB } from "@salt-ds/countries";`
+
+The following list shows every available Country Symbol alongside its country code:
+
+<Canvas>
+  <Story id="country-symbols-country-symbol--all-country-symbols-with-search" />
+</Canvas>
+
+## Metadata
```
Comment (Fercas123) [inline packages/countries/stories/country-symbol.doc.stories.mdx:54]:
```suggestion
### Metadata
```

File: .changeset/rotten-guests-sort.md [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+### Import
+
+```jsx
+import { LazyCountrySymbol } from "@salt-ds/countries;";
+```
+
+### Usage
+
+```jsx
+<LazyCountrySymbol size={1} code={AD} />
```
Comment (joshwooding) [inline .changeset/rotten-guests-sort.md:294]:
```suggestion
<LazyCountrySymbol size={1} code="AD" />
```

**Review by joshwooding — CHANGES_REQUESTED** [review]

**Pull request merged**